components/tel/Core/Inc/utils.h

```diff
@@ -0,0 +1,25 @@
+#include <stdint.h>
```

Comment:
Instead of copy-pasting this file across multiple projects, we can move it to ./libraries/ 
Multiple projects can access it using relative file path. IE #include "../../../../libraries/utils/utils.h"

Reply:
In the future, we can include the directory so we dont need to use a long file path.